org.opentosca.container.api.legacy/src/org/opentosca/container/api/legacy/resources/csar/CSARsResource.java

```diff
@@ -91,13 +91,13 @@ public CSARsResource() {
     @GET
     @Produces(ResourceConstants.LINKED_XML)
     public Response getReferencesXML() {
-        return Response.ok(this.getRefs().getXMLString()).build();
+        return Response.ok(getRefs().getXMLString()).build();
```

Comment:
api.legacy will be removed in the near future. changes in it's files are not really productive